datadog/dogshell/wrap.py

```diff
@@ -110,8 +112,10 @@ def execute(cmd, cmd_timeout, sigterm_timeout, sigkill_timeout,
     try:
         # Let's that the threads collecting the output from the command in the
         # background
-        out_reader = OutputReader(proc.stdout, sys.stdout if not buffer_outs else None)
-        err_reader = OutputReader(proc.stderr, sys.stderr if not buffer_outs else None)
+        stdout = sys.stdout if PYTHON2 else sys.stdout.buffer
+        stderr = sys.stderr if PYTHON2 else sys.stderr.buffer
```

Comment:
Let's use the existing `is_py3k()` method from `util.compat`

Reply:
Ah I didn't see this, fixed